packages/pds-ember/addon/components/pds/logomark/nomad.hbs

```diff
@@ -0,0 +1,12 @@
+<Pds::Logomark
+  ...attributes
+  data-test-logomark-nomad
+  aria-label="Nomad logomark"
+>
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <path d="M89.123 49.189v21.24L74.35 78.96l-10.338-6.409v55.429l.282.019 55.711-32.01V32.067h-1.034L89.123 49.189z"/>
+    <path d="M63.542 0L8 32.085l37.479 22.01 7.575-4.624 18.42 10.281V38.4l17.65-10.6v21.389l30.881-17.123v-.075L63.542 0z"/>
+    <path d="M56.55 68.304V89.62l-16.804 10.657V57.628l6.372-3.928-37.78-21.634-.338.019V95.99l56.012 31.99V72.552l-7.462-4.248z"/>
```

Comment:
Same question here re: coloring - one side of the cube is darker when it's in color.

Reply:
black [logomark](https://drive.google.com/file/d/1R7Gk3uGunhFFg03eML2HHm1jzx94QenW/view?usp=sharing) is solid (all same color)